modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -1,117 +1,63 @@
 # coding: utf-8
-
-"""
-Copyright 2015 SmartBear Software
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-
-    Ref: https://github.com/swagger-api/swagger-codegen
+""""
+NOTE: This file is auto generated by the swagger-codegen.
+Do not edit manually.
 """
 
 {{#models}}
 {{#model}}
 from pprint import pformat
-from six import iteritems
 
 
 class {{classname}}(object):
     """
-    NOTE: This class is auto generated by the swagger code generator program.
-    Do not edit the class manually.
+    {{#vars}}:ivar {{name}}: {{description}}
+    :type {{name}}: {{{datatype}}}
+    {{/vars}}
     """
-    def __init__(self):
-        """
-        {{classname}} - a model defined in Swagger
-
-        :param dict swaggerTypes: The key is attribute name
-                                  and the value is attribute type.
-        :param dict attributeMap: The key is attribute name
-                                  and the value is json key in definition.
-        """
-        self.swagger_types = {
-            {{#vars}}'{{name}}': '{{{datatype}}}'{{#hasMore}},
-            {{/hasMore}}{{/vars}}
+    def __init__(self{{#vars}}, {{name}}=None{{/vars}}):
+        self.swagger_type_by_variable_name = {
+            {{#vars}}'{{name}}': '{{{datatype}}}',
+            {{/vars}}
         }
-
-        self.attribute_map = {
-            {{#vars}}'{{name}}': '{{baseName}}'{{#hasMore}},
-            {{/hasMore}}{{/vars}}
+        
+        self.external_name_by_attribute_name = {
```

Comment:
The variable name `swagger_types` and `attribute_map` are also used in other programming languages (PHP, Ruby, etc). Not sure why it's necessary to rename those.